xapian-applications/omega/templates/query

```diff
@@ -103,7 +103,7 @@ $if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesiz
 </div>
 </div>
 </td>
-<td><b><a href="$html{$field{url}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b>
+<td><header><a href="$html{$field{url}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></header>
```

Comment:
I didn't know that `td` counted as a sectioning root. Cool :-)

However: this shouldn't be a `table` at all, because it's semantically a list: it should be `ol`. (The table is just for layout.) Once you've done that, you can't use a `header` here.